pages/common/django-admin.md

```diff
@@ -0,0 +1,20 @@
+# django-admin
+
+> A server-side MVT framework written in Python.
+> More information: <https://www.djangoproject.com>.
```

Comment:
Is there a more appropriate link and description, related to django-admin CLI tool specifically?

Reply:
Maybe this link is more specific to django-admin CLI tool.
https://docs.djangoproject.com/en/3.0/ref/django-admin/

Reply:
Yup, that's nice.
```suggestion
> More information: <https://docs.djangoproject.com/en/3.0/ref/django-admin/>.
```